specification/synapse/data-plane/Microsoft.Synapse/preview/2020-10-01-preview/monitoring.json

```diff
@@ -0,0 +1,651 @@
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SQLPoolName"
+          },
+          {
+            "$ref": "#/parameters/SQLDatabaseName"
+          },
+          {
+            "$ref": "#/parameters/SQLStreamingJobName"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SQLStreamingJobAuthoringLogListViewResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SparkJobListViewResponse": {
+      "type": "object",
+      "properties": {
+        "nJobs": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "sparkJobs": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SparkJob"
+          },
+          "x-nullable": true
+        }
+      }
+    },
+    "SparkJob": {
+      "type": "object",
+      "properties": {
+        "state": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "submitter": {
+          "type": "string"
+        },
+        "compute": {
+          "type": "string"
+        },
+        "sparkApplicationId": {
+          "type": "string"
+        },
+        "livyId": {
+          "type": "string"
+        },
+        "timing": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sparkJobDefinition": {
+          "type": "string",
+          "x-nullable": true
+        },
+        "pipeline": {
+          "uniqueItems": false,
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SparkJob"
+          }
+        },
+        "jobType": {
+          "type": "string"
+        },
+        "submitTime": {
+          "format": "date-time",
+          "type": "string",
+          "x-nullable": true
+        },
+        "endTime": {
+          "format": "date-time",
+          "type": "string",
+          "x-nullable": true
+        },
+        "queuedDuration": {
+          "type": "string"
+        },
+        "runningDuration": {
+          "type": "string"
+        },
+        "totalDuration": {
+          "type": "string"
+        }
+      }
+    },
+    "SqlQueryStringDataModel": {
+      "type": "object",
+      "properties": {
+        "query": {
+          "type": "string"
+        }
+      }
+    },
+    "SQLStreamingJobListViewResponse": {
```

Comment:
>SQLStreamingJobListViewResponse [](http://example.com/codeflow?start=5&length=31)

Can we add some descriptions to these models and properties inside these models to help SDK users to understand them better?